frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/namespace-store-pages/multi-namespace-store.tsx

```diff
@@ -23,22 +23,26 @@ export const MultiNamespaceStorePage: React.FC<MultiNamespaceStoreProps> = React
     const openModal = () => NamespaceStoreModal({ namespace });
     const [selectedCount, setSelectedCount] = React.useState(state.readNamespaceStore.length);
     const [enabledItems, setEnabledItems] = React.useState([]);
+    const [selectedItem, setSelectedItem] = React.useState('');
```

Comment:
React.useState<string>('');

Reply:
Do you mean React.useState(''"); with double quotes..... It already is React.useState('');(single quotes)

Reply:
Ah, i mean to say use React.useState<string>('');

Reply:
No problem :-) Lets keep it the same... typescript will automatically infer the type, no need to specify it